src/withLocation.tsx

```diff
@@ -1,12 +0,0 @@
-import React from 'react'
```

Comment:
This was only used by `AppInitializer`, which can now just call the `useLocation` hook itself